### DISCUSSION TITLE
Truthy / Falsy 한 값이 무엇인가요?

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#12] [타입변환과 단축 평가]
Truthy / Falsy 한 값이 무엇인가요?

**Comment by publdaze** [comment]
> - Truthy / Falsy 한 값이 무엇인가요?
- Falsy값은 false로 평가되는 값. false, 빈문자열, +0, -0, NaN, null, undefined
- Truthy값은 Falsy값이 아닌 true로 평가되는 값. 비어있지 않은 문자열, 0이 아닌 숫자, 객체, 배열 등

**Comment by pipisebastian** [comment]
>Truthy / Falsy 한 값이 무엇인가요?

자바스크립트 엔진은 boolean이 아닌 값들을 true, false대신 Truthy, Falsy로 구분합니다. 이는 조건식처럼 boolean으로 평가되어야할때, 
Truthy, Falsy가 true, false로 암묵적 타입 변환이 되어서 판별되는데요.

Falsy : false, undefined, null, 0, -0, NaN, ‘’(빈 문자열) ⇒ false 로 평가된다.
